custom-routes-vis/src/main/webapp/index.html

```diff
@@ -41,18 +41,18 @@ <h5>Destination Coordinates:</h5>
           <option value="compute-routes">ComputeRoutes API(V1)</option>
           <option value="compute-routes-alpha">ComputeCustomRoutes API(V1 alpha)</option>
         </select>
-        <input id="api-key" class="form-control" placeholder="API Key (Defaults to use Directions API Key)"></input>
+        <input id="api-key" class="form-control" placeholder="API Key (Optional, Default Key Exists)"></input>
         <div id="rate-card-data" class="hidden">
           <div class="row">
             <label for="cost-per-minute" class="col-sm-5">Cost Per Minute:</label>
             <div class="col-sm-7">
-              <input type="number" id="cost-per-minute" class="form-control"></input>
+              <input type="number" id="cost-per-minute" class="form-control" placeholder="Required"></input>
             </div> 
           </div>
           <div class="row">
             <label for="cost-per-km" class="col-sm-5">Cost Per km:</label>
             <div class="col-sm-7">
-              <input type="number" id="cost-per-km" class="form-control"></input>
+              <input type="number" id="cost-per-km" class="form-control" placeholder="Required"></input>
             </div>
           </div>
           <div class="row">
```

Comment:
Can we make this control value default to false? Most of our test cases disable the tolls.

(Sorry somehow it only allows me to add the comment here. I guess it's +/- 3 lines around the change?)

Reply:
done. I usually click on the plus sign on the left of each line in the "files changed" tab to add a comment